test/java/org/opendatakit/briefcase/ui/SwingTestRig.java

```diff
@@ -0,0 +1,74 @@
+package org.opendatakit.briefcase.ui;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.CopyOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.opendatakit.briefcase.model.BriefcasePreferences;
+import org.opendatakit.briefcase.util.FileSystemUtils;
+
+public class SwingTestRig {
+  private static final Log log = LogFactory.getLog(SwingTestRig.class);
+
+  public static void prepareBriefcaseStorageFolder() {
+    try {
+      BriefcasePreferences.setBriefcaseDirectoryProperty(Files.createTempDirectory("briefcase_test").toString());
+      File briefcaseFolder = new StorageLocation().getBriefcaseFolder();
+      boolean mkdirs = briefcaseFolder.mkdirs();
```

Comment:
Consider a more meaningful name, like `dirsMade` or `made` or `created` or something?
(15 seconds later)  Or inline the variable.